gradle.properties

```diff
@@ -1,3 +1,4 @@
+#latest release version with "-SNAPSHOT" prefix
```

Comment:
Unfortunately, now it is not always possible to know in advance what the version of the next release will be.

This is done to check the version of the latest release in the examples. It might be worth adding an additional property.